CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing
 
 We'd love to accept your code patches! However, before we can take them, we
-have to jump a couple of legal hurdles.
+have to jump through a couple of legal hurdles.
```

Comment:
Thx for the fix, but I think this is a mixed metaphor. One would either "jump through hoops" or "jump over hurdles".